src/2013.ipynb

```diff
@@ -1089,8 +826,8 @@
       "Secondary Area: Network Applications\n",
       "Primary OS: Windows\n",
       "Primary Technology: C/C++\n",
-      "\n",
-      "secondary_area ANOMALY: 27 Project Title: Internet Controlled Security Robot\n",
+      " []\n",
+      "primary_os ANOMALY: 27 Project Title: Internet Controlled Security Robot\n",
```

Comment:
All of these anomaly warnings are just missing fields in projects themselves.
You can actually check by searching for "primary_os ANOMALY" in this notebook  (how many OS fields are missing), and then for "Primary OS" in the txt file (how many OS fields are there). They should add up to the total number of projects.
Same for the secondary tech, primary area, secondary area, etc. fields.